packages/route/src/Domain/Model/Route.php

```diff
@@ -16,6 +16,9 @@
  */
 class Route
 {
+    /** @internal */
+    public const HISTORY_RESOURCE_KEY = 'history';
```

Comment:
maybe `route_history` better match in case of a resource key

Reply:
Yes, snake_case is used for resource keys.

I agree and would also go with something like `route_history`. It is not only more self-explanatory, but we also minimize the chance of colliding with resource keys of other custom entities.